### ISSUE TITLE
Regarding the number of iterations within each epoch

### ISSUE THREAD

**Issue by wenouyang** [EdwardTyantov/ultrasound-nerve-segmentation#4]
Hi Edward,

Regarding the the main optimization step 

`model.fit(
                   x_train, [y_train, y_train_2], 
                   validation_data=(x_valid, [y_valid, y_valid_2]),
                   batch_size=128, nb_epoch=50,
                   verbose=1, shuffle=True,
                   callbacks=[model_save_best, model_checkpoint, early_s]
                   ) 
`

How many iterations within each epoch will it be included? Here, we have about 5635 pairs of training images and training mask images. If batch_size=64, should it be 5635/32 iterations?
I did not find a place where this parameter of training_iters are setup. Will it be handled automatically by model.fit?

Thank you very much.

**Comment by EdwardTyantov** [comment]
Hi, wenouyang. The train data is automatically splitted by batch_size, so in case of batch_size=64 it will be 89 iterations (88 full batches, and final incomplete one)

**Comment by wenouyang** [comment]
Hi Edward,

Thanks for the reply.

wenouyang
